src/app/modules/shared/components/details-fields/details-fields.component.ts

```diff
@@ -257,7 +246,10 @@ export class DetailsFieldsComponent implements OnChanges, OnInit {
   onGoingToWorkOnThisChange(event: any) {
     this.goingToWorkOnThis = event.currentTarget.checked;
     if (!this.goingToWorkOnThis) {
-      this.entryForm.patchValue({ end_hour: formatDate(new Date(), 'HH:mm:ss', 'en') });
+      this.entryForm.patchValue({
```

Comment:
There is an opportunity to improve our coverage, testing this line according to Codecov. :)